src/js/angular/core/services/jwt-auth.service.js

```diff
@@ -398,7 +397,7 @@ angular.module('graphdb.framework.core.services.jwtauth', [
                 return this.isAdmin() || this.isRepoManager();
             };
 
-            this.canWriteRepo = function (repo) {
+            this.canWriteRepo = function (repo, graphql = false) {
```

Comment:
You have the default graphql=false at the checkRights method so you don't need to put it everywhere, just in case

Reply:
removed as redundant